src/west/app/project.py

```diff
@@ -844,6 +844,53 @@ def do_run(self, args, user_args):
                 failed.append(project)
         self._handle_failed(args, failed)
 
+        # List projects not tracked by manifest.
+        projects = [project.abspath for project in self._cloned_projects(args, only_active=True)]
+        topdir = Path(self.topdir)
+        untracked = []
+
+        self.do_find_untracked(topdir, projects, untracked)
+
+        ignored_config = self.config.get('status.ignore')
+        ignored = ignored_config.split(',') if ignored_config else []
+        # Ignore .west folder in untracked projects
+        ignored.append('.west/')
+        for ignore in ignored:
+            try:
+                untracked.remove(ignore)
+            except:
+                pass
+
+        if len(untracked) > 0:
+            self.banner('untracked:')
+            for project in untracked:
+                self.inf(textwrap.indent(f'{project}', ' ' * 4))
+
+    def is_manifest_subfolder(self, path, projects):
```

Comment:
Either I don't understand this function, or the name is very confusing. Do you mean something like `is_in_some_west_project()`?

The manifest is a list of git projects, it's not the projects themselves.

Anyway you should not need such a path comparison because the directory walk should prune `west` projects as soon as it finds one?